project/Projects.scala

```diff
@@ -79,6 +79,16 @@ object Projects extends Build {
         provided(aspectJ) ++
         test(scalatest, akkaTestKit, akkaSlf4j, slf4Jul, slf4Log4j, logback))
 
+  lazy val kamonAkkaHttp = Project("kamon-akka-http-server", file("kamon-akka-http-server"))
```

Comment:
The project can be just `kamon-akka-http`, as both server and client instrumentation and goodies will end up in the very same module.